packages/thirdweb/src/wallets/interfaces/wallet.ts

```diff
@@ -42,6 +42,9 @@ export type Wallet<TWalletId extends WalletId = WalletId> = {
   getConfig: () => CreateWalletArgs<TWalletId>[1];
 };
 
+// Symbols to track account origins internally for downstream adapters
+// eslint-disable-next-line better-tree-shaking/no-top-level-side-effects
+export const accountKeySymbol: unique symbol = Symbol("key");
```

Comment:
instead of exporting this directly can we export a (still marked as `@internal`) function to access the key (which can return Hex | undefined)? - basically I don't want anyone to accidentally start using this symbol for anything but this specific purpose

Reply:
Copy that